shell/models/cluster.js

```diff
@@ -14,6 +14,7 @@ export const ANNOTATIONS_CONTAINS_PROTECTED = [
   'k3s.io',
   'kubernetes.io',
   'k3s.io',
+  'rancher.io'
```

Comment:
i'm trying to find out why the hell i suggested this, as it's not referenced in this PR or RFC. might have context switched and made an error. if the feature doesn't use it, it can be removed